### PR TITLE
test(e2e): added html upload tests

### PR DESCRIPTION
**Pull request by LaxSoma**
## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/e2e/cypress/page_objects/documentPropertiesPage.js [hunk 1]
```diff
@@ -119,6 +119,15 @@ export class DocumentPropertiesPage extends Page {
 	    cy.get('p:nth-child(3) strong').should('have.text','Unpublished:');
 
 	}
+<<<<<<< HEAD
```
Comment (francescobrenci-AR) [inline apps/e2e/cypress/page_objects/documentPropertiesPage.js:122]: there is an unresolved GIT conflict below - please fix

**Comment by francescobrenci-AR** [comment]
IT seems there is a lot of repetition on those three new files.

It makes more sense to add the new cases to the existing documentUpload.specs file and avoid duplicated tests

File: apps/e2e/cypress/e2e/back-office-applications/uploadHtmlFiles.spec.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+		searchResultsPage.clickLinkByText('Update project information');
+		searchResultsPage.clickLinkByText('Project documentation');
+		searchResultsPage.clickLinkByText('Project management');
+		fileUploadPage.clickLinkByText('View/Edit properties')
+		documentPropertiesPage.enterIncorrectDocumentRefNumber(caseRef);
+		documentPropertiesPage.validateDocumentErrorMessage();
+	});
+
+
+	it('Inspector user should not be able to upload a html document to a case', () => {
```
Comment (francescobrenci-AR) [inline apps/e2e/cypress/e2e/back-office-applications/uploadHtmlFiles.spec.js:73]: this has already being tested

File: apps/e2e/cypress/e2e/back-office-applications/uploadMP3Files.spec.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+		searchResultsPage.clickLinkByText('Update project information');
+		searchResultsPage.clickLinkByText('Project documentation');
+		searchResultsPage.clickLinkByText('Project management');
+		fileUploadPage.clickLinkByText('View/Edit properties')
+		documentPropertiesPage.enterIncorrectDocumentRefNumber(caseRef);
+		documentPropertiesPage.validateDocumentErrorMessage();
+	});
+
+
+	it('Inspector user should not be able to upload a html document to a case', () => {
```
Comment (francescobrenci-AR) [inline apps/e2e/cypress/e2e/back-office-applications/uploadMP3Files.spec.js:73]: this has already being tested

File: apps/e2e/cypress/e2e/back-office-applications/uploadVideoFiles.spec.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+		searchResultsPage.clickLinkByText('Update project information');
+		searchResultsPage.clickLinkByText('Project documentation');
+		searchResultsPage.clickLinkByText('Project management');
+		fileUploadPage.clickLinkByText('View/Edit properties')
+		documentPropertiesPage.enterIncorrectDocumentRefNumber(caseRef);
+		documentPropertiesPage.validateDocumentErrorMessage();
+	});
+
+
+	it('Inspector user should not be able to upload a video document to a case', () => {
```
Comment (francescobrenci-AR) [inline apps/e2e/cypress/e2e/back-office-applications/uploadVideoFiles.spec.js:73]: this has already being tested

File: apps/e2e/cypress/e2e/back-office-applications/uploadVideoFiles.spec.js [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+		const docRefer=Cypress.env("DocRef")
+		var splitRef=docRefer.split(' ')[3]
+		documentPropertiesPage.enterDocumentRefNumber(splitRef);
+		})
+		documentPropertiesPage.validateTranscriptValue();
+
+
+	});
+
+	it('Case Team Admin user enters incorrect document reference number then error will displayed ', () => {
```
Comment (francescobrenci-AR) [inline apps/e2e/cypress/e2e/back-office-applications/uploadVideoFiles.spec.js:57]: this test is repeated three times

**Review by francescobrenci-AR — CHANGES_REQUESTED** [review]

**Comment by LaxSoma** [comment]
Not required, hence closing this PR

**Pull request closed without merging**